src/react-apps/applications/altinn-app-frontend/src/features/form/data/actions/fetch/index.ts

```diff
@@ -5,31 +5,39 @@ export interface IFetchFormData extends Action {
   url: string;
 }
 
+export interface IFetchFormDataInitial extends Action {}
```

Comment:
Is there a need to extend action here? Just use action imo :)

Reply:
Nope, fixed :)